Dev/CircleCI/circleci-check.5m.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# <bitbar.title>CircleCI Check</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Florent Segouin</bitbar.author>
+# <bitbar.author.github>fsegouin</bitbar.author.github>
+# <bitbar.desc>This plugin displays the build status of repositories listed on CircleCI.</bitbar.desc>
+# <bitbar.dependencies>python</bitbar.dependencies>
```

Comment:
It'd be awesome if you could take the image you've put in the PR description and attach it here.